doc/archbuild.1.asciidoc

```diff
@@ -0,0 +1,48 @@
+archbuild(1)
+============
+
+Name
+----
+archbuild - a script to build an Arch Linux package inside a clean chroot. 
+
+Synopsis
+--------
+archbuild [options] -- [makechrootpkg args]
+
+Description
+-----------
+
+'archbuild' is a script to build an Arch Linux package. devtools includes archbuild, but you don't run this command directly. 
+It also includes these symlinks:
+
+* kde-unstable-x86_64-build 
+* multilib-testing-build 
+* testing-x86_64-build
+* extra-x86_64-build
+* multilib-build 
+* gnome-unstable-x86_64-build  
+* multilib-staging-build     
+* staging-x86_64-build
+
+The symlink used to run it will be inspected by archbuild, to determine which target you want it to use. 
```

Comment:
Maybe this should also explain the configuration file loading when you use `staging-x86_64-build`
```
devtools /usr/share/devtools/pacman-kde-unstable.conf
```